hnn_core/network.py

```diff
@@ -984,24 +1002,68 @@ def add_connection(self, src_gid, target_gid, loc, receptor,
             Synaptic delay in ms.
         lamtha : float
             Space constant.
+
+        N.B. Connections are stored in:
```

Comment:
You can have a section called "Notes" like how we have in MNE:

https://github.com/mne-tools/mne-python/blob/6ae3b22033c745cce5cd5de9b92da54c13c36484/mne/io/array/array.py#L42

Reply:
not yet addressed